README.md

```diff
@@ -1,33 +1,151 @@
+pf flow init --flow ./my_chatbot --type chat
+```
+
+<details>
+<summary><b>Setup a connection for your API key</b></summary>
+Navigate to the `my_chatbot` folder, you can find a yaml file named `openai.yaml` file, which is the definition of the connection to store your Open AI key.
+</details>
+
+> <details>
+> <summary>For Azure OpenAI key, establish the connection by running:</summary>
+>
+> ```sh
+> # Override keys and endpoint with --set to avoid yaml file changes
+> pf connection create --file ./my_chatbot/azure_openai.yaml --set api_key=<your_api_key> api_base=<your_api_endpoint>
+> ```
+> </details>
+
+For OpenAI key, establish the connection by running:
+
+
+```sh
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ./my_chatbot/openai.yaml --set api_key=<your_api_key>
+```
+
+<details>
+<summary><b>Chat with your flow</b></summary>
+In the `my_chatbot` folder, there's a `flow.dag.yaml` file that outlines the flow, including inputs/outputs, tools, nodes, etc. Note we're using the connection named `open_ai_connection` in the `chat` node.
+</details>
+
+> <details>
+> <summary>For Azure Open AI users, modify this file accordingly.</summary>
+> Replace the 'node:' section with following content and specify the 'deployment_name' to the model deployment you'd like to use.
+>
+> ```yaml
+> nodes:
+> - name: chat
+>   type: llm
+>   source:
+>     type: code
+>     path: chat.jinja2
+>   inputs:
+>     deployment_name: <your_azure_open_ai_deployment_name>
+>     max_tokens: '256'
+>     temperature: '0.7'
+>     chat_history: ${inputs.chat_history}
+>     question: ${inputs.question}
+>   api: chat
+>   connection: azure_open_ai_connection
+> ```
+>  </details>
+
+For OpenAI users, interact with your chatbot by running: (press `Ctrl + C` to end the session)
+
+```sh
+pf flow test --flow ./my_chatbot --interactive
+```
+
+Then you will see the chatbot in action:
+![interactive](examples/tutorials/quick-start/media/interactive.gif)
+
+Next Step! Continue with the **Tutorial**  👇 section to delve deeper into Prompt flow.
+
+## Tutorial 🏃‍♂️
+
+Prompt Flow is a tool designed to **facilitate high quality LLM-native apps to production**, the development process in prompt flow follows these steps: develop a flow, improve the flow quality, deploy the flow to production.
+
+### Develop your own LLM apps
+
+[Getting Started with Prompt Flow](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html): A step by step guidance to invoke your first flow run.
+
+<details>
+<summary><b>VS Code Extension</b> <img src="examples/tutorials/quick-start/media/logo_pf.png" alt="logo" width="20"/> -- a flow designer</summary>
+
+We also offer a VS Code extension for an interactive flow development experience with UI. This is a detailed walkthrough step-by-step to create your own flow from scratch and invoke your first flow run.
+
+[![vsc extension](https://img.youtube.com/vi/GmhasXd7sj4/0.jpg)](https://youtu.be/GmhasXd7sj4)
```

Comment:
It will be strange to hide this video with one click. If we want to expose something, we can put it outside directly

Reply:
fixed